kafka/src/test/java/io/atleon/kafka/KafkaLagThresholdStarterStopperTest.java

```diff
@@ -52,6 +53,48 @@ public void startStop_givenConsumerGroupHasLagAboveThreshold_expectsStopSignal()
             .verify();
     }
 
+    @Test
```

Comment:
Since the underlying implementation remains the same, and we're just now able to pass multiple consumer groups, I'd say just replace the existing tests with the ones you've added

Reply:
Removed the single group test cases except `startStop_givenConsumerGroupIsNotAssociatedWithAnyTopicPartitions_expectsStartSignal` which is distinct. LMK if you disagree.